pkg/actions/irsa/irsa.go

```diff
@@ -17,6 +17,14 @@ type Manager struct {
 	clientSet    kubeclient.Interface
 }
 
+type Action string
+
+const (
+	Create Action = "create"
+	Delete Action = "delete"
+	Update Action = "update"
+)
```

Comment:
We don't need to export these. Additionally, this can be made more readable by prefixing the action values with the enum name. 

```suggestion
type action string

const (
	actionCreate action = "create"
	actionDelete action = "delete"
	actionUpdate action = "update"
)
```